### DISCUSSION TITLE
When does the `use_live_version` parameter in Deliver has an effect?

### DISCUSSION THREAD

**Discussion by phjs** [fastlane/fastlane#19789] [General]
App Store Connect can manage two versions of the same app at a time (_live_ and _edit_), and as far as I understand the documentation for `use_live_version`, the parameter can be used to force the usage of the _live_ version.

We tried using it during the download of metadata and screenshots, but regardless of whether we set it to `true` or `false`, Deliver always downloaded the metadata/screenshots of the _edit_ version. Only the parameter `app_version` allowed us to enforce the version we wanted to download from.

Another try was to use it during upload, but this hasn't had any effect either. At this point, `edit_live` was the parameter that led to the desired result (enforce the app version to upload to).

Do we miss a use case of Deliver where it has an effect or do we use it wrong?

**Comment by via-guy** [comment]
@phjs this should be an issue, not a discussion
